function main():void {
  var x:int = 1;
  try { var x:int = 5; throw x; var y:int = x/0;}   // division by zero unreachable
  catch(y) {y = y + 2; print(y," ");}       // new y here, the one declared by catch
  print(x,"\n");
}

// 7 1
